Tasks/Barabanov/Task2/Task2.py

```diff
@@ -0,0 +1,146 @@
+import datetime
+
+while True:
+    while True:
+        input_value = input(
+            "Select items:\n1. Enter time in format 'HH:MM'.\n2. Press 'Enter' to show current time.\n"
+            "3. Enter 'exit' to close.\n"
+        )
+        try:
+            if input_value == "":
+                hh, mm = datetime.datetime.strftime(
+                    datetime.datetime.now(), "%I %M"
+                ).split(" ")
+            elif input_value.lower() == "exit":
+                exit("Program was finished by user.")
+            else:
+                hh, mm = input_value.split(":")
+
+            if (
+                len(hh) == 2
+                and len(mm) == 2
+                and 0 <= int(hh) < 24
+                and 0 <= int(mm) <= 59
+            ):
+                break
+            else:
+                raise Exception
+        except Exception:
```

Comment:
it's better to raise and catch some specific error, since something may go wrong in the block, and your catch it with this generic except